app/base_api_error.py

```diff
@@ -6,5 +6,5 @@ def __init__(self, message, error_code):
         super().__init__(message)
         self.error_code = error_code
 
-    def buildError(self):
-        return jsonify({"success": False, "message": self.message}), self.error_code
+    def build_error(code, message):
+        return BaseApiError(message, code)

```

Comment:
Why wouldn't you just call the constructor here. I liked the pattern of having a buildError function that you could call on any api error.
Building on this pattern you could add the `code`, `error` and `message` attributes on this exception. `Code` is the integer error, `error` is the error name and `message` is the description or detailed error message.
You can then kind of keep this `buildError` function (I like re-naming it `jsonify` to keep it simple) that will return a dict with the error and message and error code.

Reply:
Oh I see I think I misinterpreted what was on the design doc. Will fix